lib/oauth2/authenticator.rb

```diff
@@ -25,6 +25,10 @@ def apply(params)
         apply_basic_auth(params)
       when :request_body
         apply_params_auth(params)
+      when :tls_client_auth
+        apply_client_id(params)
+      when :private_key_jwt
+        params
```

Comment:
@SteveyblamFreeagent just straight `params`?  The comment on line 49 appears to imply that `apply_client_id` would be used for `private_key_jwt` also.

Reply:
Thanks for looking at this Peter. Straight params is expected behaviour as private_key_jwt wraps up the client_id in the assertion so it's not passed in the request body. But you're absolutely right the comment was from a previous commit where I was applying the client_id. I'll change the comment now.

Many thanks